app/models/interest.rb

```diff
@@ -2,4 +2,7 @@ class Interest < ApplicationRecord
   self.inheritance_column = :_type_disabled
   enum type: { health: 0, hobby: 1, work: 2 }
   belongs_to :user
+  scope :amount_interest_cosmhealth_young_woman, -> { where("type = 0 and name like 'cosm%'").
```

Comment:
It isn't well formatted. All scopes should return an ActiveRecord::Relation object, because you should be able to chain them.
So count method can be called in the controller.

I really don't have an idea for a good name, maybe female_interests is enough.